client/style.css

```diff
@@ -540,15 +540,20 @@ h1{
   float: right;
 }
 
-.add-question-form label, .add-answer-form label {
+.add-form label, .add-answer-form label {
   width: 10%;
   text-align: right;
   display: inline-block;
   padding-right: 1%;
 }
 
-.add-question-form input, .add-question-form textarea, .add-answer-form input, .add-answer-form textarea{
-  width: 50%
+.add-form input, .add-question-form textarea, .add-answer-form input, .add-answer-form textarea{
+  width: 70%
+}
+
+#index h2 {
```

Comment:
The id of `index` won't apply to the modal divs you have above. You could add a classname to that div along with that id, and that would grab them all.

Reply:
This was me testing the question modal and I forgot to rename it. I will rename it to align to the correct modal it is using. The other index ids I made, don't need styling added, I just need to be able to grab them.